src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -904,12 +903,15 @@ export default class Aircraft {
 
         let isExpeditingString = '';
         if (expedite) {
-            isExpeditingString = 'and expedite';
+            isExpeditingString = ' and expedite';
```

Comment:
Yeah, I thought that would be the case here.  Does the extra space at the end actually affect how its displayed in the ui?  I'd rather not have `trim` in there, though that is an option.

I'd rather leave it as-is and just have you add a comment (pretty much what you said above).